### PR TITLE
Corrects the Escape Alone text

### PR DESCRIPTION
**Pull request by Dorsisdwarf**
A while ago, Escape Alone was updated so that you could bring as many antagonists on the shuttle with you as you liked, but the description doesn't hint that this is the case. This PR fixes that

### CODE REVIEW

**Comment by Jalleo** [comment]
Silicons and pai are not counted i believe

**Comment by Incoming5643** [comment]
Silicons don't count because they're registered as dead, I'm not 100% sure about pAIs considering how in flux the pAI mob has been in over the ages

File: code/game/gamemodes/objective.dm [hunk 1]
```diff
@@ -186,7 +186,7 @@
 
 
 /datum/objective/hijack
-	explanation_text = "Hijack the emergency shuttle by escaping alone."
+	explanation_text = "Hijack the emergency shuttle by ensuring that only non-Nanotrasen entities escape alive, aboard, and not in the shuttle brig."
```
Comment (Incoming5643) [inline code/game/gamemodes/objective.dm]: I really wish there was a more terse way of phrasing this, as I'm pretty sure that's why I didn't end up changing it initially.

Comment (phil235) [inline code/game/gamemodes/objective.dm]: Agreed, the text is a bit too long and awkward.

**Comment by Iamgoofball** [comment]
why not change it to

"Hijack the shuttle and ensure that only antagonists escape on board the shuttle."

File: code/game/gamemodes/objective.dm (hunk 1 above)
Comment (ojacobson) [inline code/game/gamemodes/objective.dm]: > Hijack the emergency shuttle by making sure no Nanotrasen employees ride outside the brig.

**Comment by Jalleo** [comment]
If non antags are in the shuttle brig they arent counted btw
How about:
Hijack the shuttle; ensure all real Nanotrasen workers arent free by arrival.

**Comment by bustygoku** [comment]
Hijack the emergency shuttle by escaping alone or with other syndicate agents

**Comment by Dorsisdwarf** [comment]
Bustygoku, the problem with that is that it's not just syndies.

**Comment by Jalleo** [comment]
Best it can get really

File: code/game/gamemodes/objective.dm [hunk 2]
```diff
@@ -186,7 +186,7 @@
 
 
 /datum/objective/hijack
-	explanation_text = "Hijack the emergency shuttle by escaping alone."
+	explanation_text = "Hijack the shuttle to ensure no loyalist Nanotrasen crew escape alive and out of custody.
```
Comment (Jordie0608) [inline code/game/gamemodes/objective.dm]: Missing a closing "

**Comment by bustygoku** [comment]
We should update the lore to specify that the changelings are syndicate since they a) can get traitor objectives/gear and b) usually go on arbitrary syndicate assassination and steal missions in addition to harvesting genomes

**Comment by Jalleo** [comment]
Not required due to how its currently worded

**Pull request merged**